### DISCUSSION TITLE
noauditlog still display rule in log

### DISCUSSION THREAD

**Discussion by gin-gitaxias** [corazawaf/coraza#806] [General]
i am a bit confused with 'noauditlog' , and specially with this rule in  REQUEST-901-INITIALIZATION.conf

```
SecRule REQBODY_PROCESSOR "!@rx (?:URLENCODED|MULTIPART|XML|JSON)" \
    "id:901340,\
    phase:1,\
    pass,\
    nolog,\
    noauditlog,\
    msg:'Enabling body inspection',\
    ctl:forceRequestBodyVariable=On,\
    ver:'OWASP_CRS/4.0.0-rc1'
```
  
With this it's not supposed to show in log right ?  Because for me it does 

here is my entire main.go 
https://github.com/bunkerity/bunkerweb-plugins/blob/dev/coraza/api/main.go

Thank for any help

**Comment by jptosso** [comment]
Hey! Auditlog, unlike log, it's just a flag that will be consumed after calling tx.ProcessLogging() to write the audit log, therefore it can be overridden by another auditlog in future rules, even in phase 5.

Can you provide more context? Are you seeing the whole auditlog? the rule match in the auditlog?

Thank you!
